README.md

```diff
@@ -53,12 +68,17 @@ Return to your original seats.
 In this phase everyone is thinking like a Designer.  Design thinking also relies on iterative refinement of work to suite the Customer.  Are there ways it could be improved?  Discuss the feedback and refine/update your work if needed.
 
 ***Q1.5 Based on the feedback, how might you change your solution?***
+We will minimizes expenses by using immersive goggles instead of a full helmet.
+We will provide information about the healthy length of time to use the machine.
```

Comment:
Psychologically, you might have to think about how such a piece of technology might be abused (say if some people always want to live in the past).